src/btree/bt_compact.c

```diff
@@ -162,8 +162,8 @@ __compact_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
     WT_ADDR_COPY copy;
     WT_BM *bm;
     WT_DECL_RET;
+    WT_REF_STATE_TYPE previous_state;
```

Comment:
Suggestion: Shorten the name to `WT_REF_STATE`

Reply:
Happy to shorten, took the prompt form the `typedef enum` aliases which use this convention. Any other reviewers care to weigh in?

Reply:
I support Andrew's suggestion to shorten the name, but it's not a must have change for me.

Reply:
@agorrod Interested in your take on this.

Reply:
I would prefer to shorten it. Most of the `_TYPE` suffixes describe the type of something (e.g. `WT_BTREE_TYPE`). There are plenty of typedefs without the suffix, e.g. `WT_VERBOSE_CATEGORY`.

Reply:
The shortens have it!